apis/nucleus/src/components/listbox/components/ListBoxRowColumn.jsx

```diff
@@ -287,6 +292,20 @@ function RowColumn({ index, style, data }) {
 
   const excludedOrAlternative = () => (isAlternative(cell) || isExcluded(cell)) && checkboxes;
 
+  const isNumeric = cell?.qNum !== 'NaN' && cell?.qNum !== undefined;
+  let valueTextAlign;
+  const isAutoTextAlign = textAlign === undefined || textAlign.auto;
```

Comment:
```suggestion
  const isAutoTextAlign = !textAlign || textAlign.auto;
```
(Unless it has be `undefined` precisely, which I don't think it has to be)